app/controllers/batch_invitations_controller.rb

```diff
@@ -91,4 +96,17 @@ def grant_default_permissions(batch_invitation)
       batch_invitation.grant_permission(default_permission)
     end
   end
+
+  def batch_users_error_message(batch_user)
+    e = batch_user.errors.first
+
+    case e.attribute
+    when :email
+      "One or more emails were invalid"
+    when :name
+      "One or more names were invalid"
```

Comment:
I'm almost certainly missing something obvious here, but it doesn't look as if there's any validation on the `BatchInvitationUser#name` attribute. So I'm unclear how this case can happen.

Reply:
Sorry, I'm not sure why I typed this. I'll push the fix together with the new error message when that's ready